README.md

```diff
@@ -2,4 +2,4 @@
 
 ## Latest Release
 
-Summer '21 🌞 ⛵ 🌊
+Summer '22 🌞 ⛵ 🌊
```

Comment:
To create a PR for `feature/238`, we had to have at least one diff.  If there are no other diffs, we won't commit this change + release for 238.